zerver/webhooks/codebase/view.py

```diff
@@ -0,0 +1,83 @@
+import json
+from typing import Any, Dict, Iterable
+
+from django.http import HttpRequest, HttpResponse
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message, \
+    UnexpectedWebhookEventType
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+def get_event_type(payload: Dict[str, Any]):
+    return (payload['type'])
+
+def get_topic_type(payload: Dict[str, Any]):
+
+    if payload['type'] == "ticket_update":
+        payload = payload['payload']
+        payload = json.loads(payload)
+
+        return payload['ticket']['summary']
+    else:
+        payload = payload['payload']
+        payload = json.loads(payload)
+
+        return payload['summary']
+
+def ticket_creation(payload: Dict[str, Any], user_profile: UserProfile):
+
+    payload = payload['payload']
+    payload = json.loads(payload)
+
+    type = payload['type']['name']
+    name = payload['assignee']['name']
+    category = payload['category']
+    url = payload['url']
+
+    body = "A ticket of **[{}]({})** type and category **{}** has been created by **{}**".format(type, url, category, name)
+
+    return body
+
+def ticket_update(payload: Dict[str, Any], user_profile: UserProfile):
+    payload = payload['payload']
+    payload = json.loads(payload)
+
+    id = payload['ticket']['id']
+    type = payload['ticket']['category']
+    name = payload['user']['name']
+    url = payload['ticket']['project']['url']
+
+    body = "Ticket with ID **[{}]({})**, category **{}** has been updated by **{}**".format(id, url, type, name)
```

Comment:
A couple of things:
* I would move both body template strings to the top of the file with all uppercase names, such as `TICKET_UPDATE_TEMPLATE`. I would also use named format arguments, such as `"has been updated by {author}".format(author=author)`
* Also, I would recommend ending all messages with proper punctuation, i.e., a period. I am currently doing an audit to make sure we use proper punctuation across all of our webhooks.